src/main/java/jrm/dao/UserRepository.java

```diff
@@ -3,14 +3,25 @@
 
 import jrm.exeptions.UserNotFoundException;
 import jrm.user.User;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.CrudRepository;
 import org.springframework.stereotype.Repository;
 
+import javax.transaction.Transactional;
+import java.sql.Timestamp;
 import java.util.Optional;
 
 @Repository
 public interface UserRepository extends CrudRepository<User, Long> {
 
+    // update creation date
+    @Transactional
```

Comment:
`@Transactional` is more service layer annotation. And anyway, using it for just single update without related entities it's incorrect decision. Please remove it.

Reply:
It was found out that query does not execute without this annotation, so we've agreed don't remove it.